content/en/user-guide/cloud-pods/pods-cli/index.md

```diff
@@ -25,24 +25,80 @@ In the above syntax:
 ## Commands
 
 The following section lists the available commands for the Cloud Pods CLI.
+You can have an overview of these command by typing `localstack pod --help`:
 
-### `save`
+{{< command >}}
+<disable-copy>
+Usage: pod [OPTIONS] COMMAND [ARGS]...
+
+  Manage the state of your instance via Cloud Pods.
 
-The `save` command allows you to save a new version of a Cloud Pod. For licensed users, it provides the option to either save the Cloud Pod locally or upload it to the LocalStack platform. 
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  delete    Delete a Cloud Pod
+  inspect   Inspect the contents of a Cloud Pod This command shows the...
```

Comment:
This command description looks a bit - can we remove the part with `This command shows the ...`? (both here, and in the CLI)

Also for the `versions` command below